test/system/070-build.bats

```diff
@@ -31,4 +31,23 @@ EOF
     run_podman rmi build_test
 }
 
+@test "podman build - test RUN with priv'd command" {
+    if [[ "$PODMAN" =~ -remote ]]; then
+        if [ "$(id -u)" -ne 0 ]; then
+            skip "unreliable with podman-remote and rootless; #2972"
+        fi
+    fi
+
+    tmpdir=$PODMAN_TMPDIR/build-test
+    run mkdir -p $tmpdir || die "Could not mkdir $tmpdir"
+    dockerfile=$tmpdir/Dockerfile
+    cat >$dockerfile <<EOF
+FROM alpine 
```

Comment:
Please use $IMAGE, not alpine; we already have $IMAGE (= `alpine_labels`) cached

Reply:
fixed